[REDESIGN] Last polish

mariaschuld opened the pull request:
**Summary:**

Last PR in the redesign sequence, collecting all final changes from reviewing the homepage.

josh146 commented:
<h3>Website build</h3>
<strong>Commit:</strong> 65ad65fca314620b3354b855e726b20ac8d3ae3f

<strong>Circle build number:</strong> 562

<strong>Website zip:</strong> <a href=https://562-214003948-gh.circle-artifacts.com/0/tmp/qml_html.zip>https://562-214003948-gh.circle-artifacts.com/0/tmp/qml_html.zip</a>

<strong>Website url:</strong> <a href=https://562-214003948-gh.circle-artifacts.com/0/_build/html/index.html>https://562-214003948-gh.circle-artifacts.com/0/_build/html/index.html</a>

<em>Please double check the rendered website build to make sure everything is correct.</em>

co9olguy commented on demonstrations/tutorial_embeddings_metric_learning.py:
```diff
@@ -135,8 +136,8 @@ def QAOAEmbedding(features, weights, wires):
-# .. note:: Instead of using the hand-coded ``QAOAEmbedding()`` function, PennyLane contains
-#           a built-in template. To use it, simply replace the cell above
+# .. note:: Instead of using the hand-coded ``QAOAEmbedding()`` function, PennyLane provides
+#           a built-in :func:`~.QAOAEmbedding` template. To use it, simply replace the cell above
```
Is the goal here to link to the actual template? Currently does not resolve (but might just be a circleCI thing @josh146?)

josh146 replied:
It should resolve! @mariaschuld, you might have to link to the full path, i.e.,
```rest
:func:`QAOAEmebedding <pennylane.templates.QAOAEmbedding>`
```
(I forget if its template or templates)

josh146 replied:
As a side note, I'm surprised this notation works. I've only ever seen 

```rest
.. note::

    Indentation used like
    this.
```

mariaschuld replied:
This is what the automatic python-to-jupyter conversion formats it as.

I used it because most demos will have the formatting, and it is consistent throughout the file.

co9olguy commented on demonstrations/tutorial_embeddings_metric_learning.py:
```diff
@@ -311,9 +312,9 @@ def cost(weights, A=None, B=None):
+# .. note:: You can alternatively use the utility function :func:`~.pennylane.init.qaoa_embedding_normal`
 #           to conveniently generate the correct shape of ``init_pars_quantum`` for
-#           the ``QAOAEmbedding``. Import it with the statement
+#           :func:`~.QAOAEmbedding`. Import it with the statement
```
same point as above

co9olguy commented on glossary.rst:
```diff
@@ -5,6 +5,8 @@
 Key Concepts
 ============
 
+Here you find a growing glossary of key concepts around the training of variational circuits.
```
```suggestion
Here you can find a growing glossary of key concepts for quantum machine learning.
```

co9olguy replied:
(since "variational ciruits" could already be a technical term that someone consulting a "Key Concepts" page may not know)

mariaschuld replied:
Good point

co9olguy commented on glossary/variational_circuit.rst:
```diff
@@ -69,9 +68,11 @@ Building the circuit
-This allows us to convert *classical information* (the values :math:`\mathbf{\theta}`)
-into *quantum information* (the quantum state :math:`U(x;\mathbf{\theta})|0\rangle`).
+The variational parameters :math:`\theta`, possibly together with an additional set of non-adaptable
+gate parameters :math:`x`, enter the quantum circuit as arguments for the circuit's gates.
```
```suggestion
parameters :math:`x`, enter the quantum circuit as arguments for the circuit's gates.
```

co9olguy commented on glossary/circuit_ansatz.rst:
```diff
@@ -10,11 +10,11 @@ In the context of variational circuits, an *ansatz* usually describes a subrouti
 
-Many variational circuit architectures have been proposed by the quantum computing community [#]_. The strength
-of an architecture varies depending on the desired use-case, and it is not always clear what makes a good ansatz.
+Many variational circuit ansaetze have been proposed by the quantum computing community [#]_. The strength
```
I wonder if we can add a footnote to make clear that this is the plural form of the German word ansatz (otherwise someone might thin it's a typo?)

mariaschuld replied:
I thought about it....In talks people always stumble over this one, and it was frankly one of the reasons to talk about "architectures" before.

Will add an explanation, I think the German plural is the most common still.

co9olguy commented on glossary/circuit_ansatz.rst:
```diff
@@ -151,8 +151,8 @@ in the ground state of :math:`A` and adiabatically evolves to the ground state o
-Tensor network architectures
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Tensor network ansatz
+~~~~~~~~~~~~~~~~~~~~~
```
Would be great to incorporate the new broadcast patterns into this page eventually :smiley:

mariaschuld replied:
Yes, this page is at the moment somewhat orthogonal to how we think about templates - more following the literature. One day one could rewrite it starting with the idea of broadcasting and constructing an ansatz from it.

co9olguy commented:
@josh146 @mariaschuld What have we decided to do going forward with the reference/citation formatting on these pages?

josh146 commented:
Main navigation cards are now 4 to a row (and slightly wider)

mariaschuld commented:
> @josh146 @mariaschuld What have we decided to do going forward with the reference/citation formatting on these pages?

I think we should keep referencing away from anything outside of the glossary, but definitely use it here! A lot of the content is rather scientific and written for researchers...

mariaschuld commented:
> Main navigation cards are now 4 to a row (and slightly wider)

When I build it is still breaking the line?

![Screenshot from 2020-02-27 09-19-00](https://user-images.githubusercontent.com/23052011/75421160-5101af80-5942-11ea-8349-32e60e08fd33.png)

mariaschuld commented on demonstrations/tutorial_isingmodel_PyTorch.py:
```diff
@@ -4,11 +4,17 @@
 3-qubit Ising model in PyTorch
 ==============================
 
```
Just breaking the lines here for better readability.

mariaschuld commented:
I changed all ``:cite:`` instances and checked every new link. Deleted the ``zreferences`` and bib files.

The pennylane repo does not have any :cite: instances either. 

Is bibtex thereby discarded or do we have to delete anything else @josh146 ?

josh146 commented:
> Is bibtex thereby discarded or do we have to delete anything else @josh146 ?

That should be it! References to `bibtex` can now be deleted, e.g., `sphinx-bibtex` in `requirements.txt`, in `conf.py`.
